context/Cargo.toml

```diff
@@ -12,12 +12,23 @@ tempfile = "3.2.0"
 [dependencies]
 anyhow = "1.0.44"
 chrono = "0.4.33"
-gix = { version = "0.63.0", default-features = false, features = [] }
-openssl = { version = "0.10.66", features = ["vendored"] }
+gix = { version = "0.63.0", default-features = false, features = [
+], optional = true }
+js-sys = { version = "0.3.70", optional = true }
+openssl = { version = "0.10.66", features = ["vendored"], optional = true }
+pyo3 = { version = "0.20.0", optional = true }
 quick-junit = "0.5.0"
 quick-xml = "0.36.2"
 regex = { version = "1.10.3", default-features = false, features = ["std"] }
 serde = { version = "1.0.130", default-features = false, features = ["derive"] }
 serde_json = "1.0.68"
 speedate = "0.14.4"
 thiserror = "1.0.63"
+wasm-bindgen = { version = "0.2.84", optional = true }
+
+[features]
+default = ["git-access"]
+git-access = ["dep:gix", "dep:openssl"]
+bindings = []
+pyo3 = ["bindings", "dep:pyo3"]
+wasm = ["bindings", "dep:wasm-bindgen", "dep:js-sys"]
```

Comment:
`gix` cannot be compiled to wasm since it depends on a unix OS and additionally, we don't need to include bindings implementations for the CLI, so features have been added to conditionally include bindings and compatibility.